platform/glfw/main.cpp

```diff
@@ -167,9 +165,10 @@ int main(int argc, char *argv[]) {
 
     // Load style
     if (style.empty()) {
-        const char *url = getenv("MAPBOX_STYLE_URL");
+        const char* url = getenv("MAPBOX_STYLE_URL");
         if (url == nullptr) {
-            mbgl::util::default_styles::DefaultStyle newStyle = mbgl::util::default_styles::orderedStyles[0];
+            mbgl::util::default_styles::DefaultStyle newStyle =
+                mbgl::util::default_styles::orderedStyles[0];
             style = newStyle.url;
             view->setWindowTitle(newStyle.name);
         } else {
```

Comment:
While all of the formatting changes in main.cpp are good, they aren't related to the pull request. If it's not too complicated, removing them might increase the reviewability.